app/models/vcap_application.rb

```diff
@@ -0,0 +1,9 @@
+class VCAPApplication
+  def self.application_uris
+    vcap_application['application_uris']
```

Comment:
TODO: encapsulate environment logic here (move it out of Presenter::Auction#url)